src/dmd/doc.d

```diff
@@ -3871,33 +3881,55 @@ private void highlightText(Scope* sc, Dsymbols* a, Loc loc, OutBuffer* buf, size
 
         case '`':
             {
-                if (inBacktick)
+                const iAfterDelimiter = skipChars(buf, i, "`");
+                int count = cast(int) (iAfterDelimiter - i);
```

Comment:
`const`?

Reply:
Why `int`?

Reply:
Made const and moved the cast to int for when it gets assigned to `inBacktick`.